<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_SelectAll"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_About_program"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/new</normaloff>:/actions/new</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/open</normaloff>:/actions/open</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/save</normaloff>:/actions/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/undo</normaloff>:/actions/undo</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/redo</normaloff>:/actions/redo</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/cut</normaloff>:/actions/cut</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/copy</normaloff>:/actions/copy</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/paste</normaloff>:/actions/paste</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="action_SelectAll">
   <property name="text">
    <string>Select &amp;all</string>
   </property>
  </action>
  <action name="action_About_program">
   <property name="text">
    <string>&amp;About program</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/preferences</normaloff>:/actions/preferences</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_Copy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_Cut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>action_Undo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>action_Redo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>modificationChanged(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setWindowModified(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Paste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Undo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_SelectAll</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Redo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Cut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Copy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
